src/containers/login/login.connector.js

```diff
@@ -2,8 +2,8 @@ import { connect } from "react-redux";
 import { signInWithGoogle } from "store/thunks/users";
 import Login from "./login.component";
 
-const mapDispatchToProps = (dispatch) => ({
-  signInWithGoogleThunk: () => dispatch(signInWithGoogle()),
-});
+// const mapDispatchToProps = (dispatch) => ({
+//   signInWithGoogleThunk: () => dispatch(signInWithGoogle()),
+// });
 
-export default connect(null, mapDispatchToProps)(Login);
+export default connect(null, null)(Login);
```

Comment:
This whole chunk can be now replaced with `export default Login`

Reply:
Can't this just all be deleted?